Doc/library/shutil.rst

```diff
@@ -657,6 +657,13 @@ provided.  They rely on the :mod:`zipfile` and :mod:`tarfile` modules.
    registered for that extension.  In case none is found,
    a :exc:`ValueError` is raised.
 
+   .. warning::
+
+      Never extract archives from untrusted sources without prior inspection.
+      It is possible that files are created outside of *path*, e.g. members
+      that have absolute filenames starting with ``"/"`` or filenames with two
```

Comment:
Does this apply to Windows as well, e.g. absolute paths like `c:\\...` and then should this be mentioned here and in `tarfile.extractall()` doc?

Reply:
This also applies to Windows. I think the comment was just giving some examples, not meaning to be comprehensive. I believe that `tar` uses `/` anyway instead of `\` for Windows as well in any case.

Reply:
I thought examples make it look like it applies only to Posix, but of course `..` does apply to Windows. I think it's fine as it is, you can ignore my comment.